src/app/community/_components/PostCardList.tsx

```diff
@@ -23,29 +21,39 @@ interface CommunityPageProps {
 }
 
 export default function PostCardList({ searchParams, initialData }: CommunityPageProps) {
+  const queryClient = useQueryClient();
+
   const getAllCommunityParams: CommunityParamsType = {
     sort: searchParams.sort || 'new',
     page: searchParams.page || '0',
     size: searchParams.size || '16',
   };
 
-  const { data: communityData, isLoading } = useQuery({
+  const {
+    data: communityData,
+    isLoading,
+    refetch,
+  } = useQuery({
     queryKey: ['postCardsList', searchParams],
     queryFn: () => getAllCommunityPost(getAllCommunityParams),
   });
 
+  const handleUpdateCardlist = () => {
+    refetch();
+    queryClient.invalidateQueries({
+      queryKey: ['post'],
+    });
```

Comment:
이런 식으로 refetch를 사용 안하고 invalidateQueries로 한번에 처리하는 것과 차이가 있을까요?(단순 궁금)
```suggestion
    queryClient.invalidateQueries({
      queryKey: ['post', ['postCardList', searchParams]],
    });
```

Reply:
우ㅜㅇㅁ,, 차이는 별로 없는 것 같아여,, 😊